k8s/service_controller/server/mem/in_mem_service_storage.go

```diff
@@ -79,8 +79,14 @@ func (s *InMemServiceStorage) AddBroker(broker *model.ServiceBroker, catalog *mo
 	return nil
 }
 
-func (s *InMemServiceStorage) DeleteBroker(string) error {
-	return fmt.Errorf("Not implemented yet")
+func (s *InMemServiceStorage) DeleteBroker(id string) error {
```

Comment:
What happens to the instances/bindings associated with this broker when it's gone? Do we need to clear anything associated with the broker?